app/src/main/res/layout-land/message_activity.xml

```diff
@@ -0,0 +1,197 @@
+<?xml version = "1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Useless parent layout. Remove this

Reply:
No it is not useless..

Reply:
Can you please explain that why it is useless?